core/src/main/scala/org/apache/spark/scheduler/dynalloc/ExecutorMonitor.scala

```diff
@@ -365,11 +366,14 @@ private[spark] class ExecutorMonitor(
       } else {
         metrics.exitedUnexpectedly.inc()
       }
-      logInfo(s"Executor ${event.executorId} is removed. Remove reason statistics: (" +
-        s"gracefully decommissioned: ${metrics.gracefullyDecommissioned.getCount()}, " +
-        s"decommision unfinished: ${metrics.decommissionUnfinished.getCount()}, " +
```

Comment:
@gengliangwang 
Unfortunately, after `correcting` the typo from `decommision` to `decommission`, it seems that GA has been broken. I have submitted a PR to solve this minor issue.
1.https://github.com/panbingkun/spark/actions/runs/9232744348/job/25406127982
<img width="925" alt="image" src="https://github.com/apache/spark/assets/15246973/4e71598c-22f3-4fd2-afba-fd91ddce5f55">

2.A pr to fix issue:
https://github.com/apache/spark/pull/46746/files